instruqt-tracks/terraform-basics-aws/track.yml

```diff
@@ -0,0 +1,472 @@
+  assignment: |-
+    Run the **terraform plan** command:
+
+    ```
+    terraform plan
+    ```
+
+    When you run this command terraform will prompt you to enter the `prefix` variable.
+
+    Enter a short string of lower-case letters and/or numbers. You can use your own name here if you wish.
+
+    **Keep your prefix string all lower case, and between 5-12 characters long. Do not use an underscore in your prefix.**
+
+    The prefix will become part of your application hostname, and therefore must be DNS-compliant. Valid characters for hostnames are ASCII(7) letters from a to z, the digits from 0 to 9, and the hyphen (-).  A hostname may not start with a hyphen.
+  notes:
+  - type: text
+    contents: |-
+      `terraform plan` allows you to preview any changes to your environment in a safe way.
+
+      This can help you identify any unexpected changes before you deploy them, not after they are already built.
+  tabs:
+  - title: Code Editor
+    type: service
+    hostname: workstation
+    port: 8080
+  - title: Shell
+    type: terminal
+    hostname: workstation
+  difficulty: basic
+  timelimit: 300
+- slug: terraform-variables
+  id: rocxqntdh4vu
+  type: challenge
+  title: "\U0001F39B️ Working with Terraform Variables"
+  teaser: Terraform variables allow you to customize your infrastructure without editing
+    any code. You can use the same terraform code to deploy dev, staging and production
+    but with different variables.
+  assignment: |-
+    We've created a **terraform.tfvars** file for you to edit. This is a special file where you can set any variables you wish to change from their defaults.
+
+    Variables are *defined* in the **variables.tf** file. They are *set* in the **terraform.tfvars** file.
+
+    In terraform all variables must be declared (with or without an optional default value) before you can use them.
+
+    Open the **terraform.tfvars** file and set your prefix variable there. Save the file.
+
+    Now run `terraform plan` again. This time you won't have to enter your prefix manually.
+  notes:
+  - type: text
+    contents: The **terraform.tfvars** file is a convenient place for users to configure
+      their variables.
+  tabs:
+  - title: Code Editor
+    type: service
+    hostname: workstation
+    port: 8080
+  - title: Shell
+    type: terminal
+    hostname: workstation
+  difficulty: basic
+  timelimit: 300
+- slug: terraform-add-a-variable
+  id: lrtbjcbhijkt
+  type: challenge
+  title: "\U0001F5FC Change Your Location"
+  teaser: Terraform is flexible enough to deploy infrastructure anywhere in the world.
+    You can easily provision your applications in different geographical regions by
+    simply changing a single variable.
+  assignment: |-
+    In one of the previous challenges we set our *prefix* variable.
+
+    Choose the AWS region nearest to you and set the location variable in your **terraform.tfvars** file. You may need to close and re-open the file if it appears blank.
```

Comment:
The variable is called `region` and not `location`. It should either be changed in the code, or changed in the instruqt instructions

Reply:
Let's stick with region, since that is the nomenclature used by AWS.